src/driver_client/dclient.rs

```diff
@@ -38,11 +38,13 @@ pub trait DriverPrimitive<T, P, I, O> {
     fn initialize(&self, param: P) -> Result<()>;
     /// The `set_data` method sets the input data for the driver primitive.
     fn set_data(&self, input: I) -> Result<()>;
+    /// The `start_process` method starts the driver after setting all controls and data.
+    fn start_process(&self, param: Option<usize>) -> Result<()>;
```

Comment:
"start_process" is very hard to understand, what process does it start? what is the implementation scope of this method?

Reply:
This is mainly done because of NTT - there are separate registers that start the process of calculations. In MSM it is possible to mark it as a set register which push to queue the next calculation
 
 If you have idea of better name, I'm open to discuss